fsw-react-calculator/src/Components/Calculator.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import Buttons from "./Buttons";
+import "../calculator.css"
+
+class Calculator extends React.Component {
+  state = { display: "0", userNum1: "", userNum2: "", operator: "", justEvaluated: false, enterSecondNum: false};
+
+  enterNum = (num) => {
+    this.setState((prevState) => {
+      let newDisplay;
+      if (prevState.display === "0" || prevState.justEvaluated || prevState.enterSecondNum) {
+        newDisplay = num;
+     
+      } else  {
+        newDisplay = prevState.display + num;
+      }
+      return {
+        display: newDisplay,
+        userNum2: newDisplay,
+        justEvaluated: false,
+        enterSecondNum: false
+      };
+    });
+  };
+
+  addition = (addition) => {
+    this.setState((prevState) => {
+      return {
+        userNum1: prevState.display,
+        operator: addition,
+        enterSecondNum: true
+      };
+    });
+  };
+  subtraction = (subtraction) => {
+    this.setState((prevState) => {
+      return {
+        userNum1: prevState.display,
+        operator: subtraction,
+        enterSecondNum: true
+      };
+    });
+  };
+  multiplication = (multiplication) => {
+    this.setState((prevState) => {
+      return {
+        userNum1: prevState.display,
+        operator: multiplication,
+        enterSecondNum: true
+      };
+    });
+  };
+  division = (division) => {
+    this.setState((prevState) => {
+      return {
+        userNum1: prevState.display,
+        operator: division,
+        enterSecondNum: true
+      };
+    });
+  };
+
+  equals = () => {
+    this.setState((prevState) => {
+      let total = 0;
+      if (prevState.operator === "addition") {
```

Comment:
A switch statement would be better suited for this conditional. 
switch(operation) {
 case: "+":

Reply:
Successfully converted conditional to switch statement